data/SiteConfig.js

```diff
@@ -1,7 +1,7 @@
 const config = {
   siteTitle: "Terasology", // Site title. Same as Short site title for home screen (PWA). Preferably should be under 12 characters to prevent truncation.
   siteTitleAlt: "Terasology Project | An Open Source Voxel World", // Alternative site title for SEO.
-  siteLogo: "/logos/logo.png", // Logo used for SEO and manifest.
```

Comment:
Keep as PNG, the JPG transformation looks weird and has big artifacts and is missing transparency.